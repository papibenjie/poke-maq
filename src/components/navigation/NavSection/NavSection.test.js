import React from 'react'
import renderer from 'react-test-renderer'
import { NavSection } from '.'

it('NavSection: default', () => {
  const component = renderer.create(<NavSection />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
