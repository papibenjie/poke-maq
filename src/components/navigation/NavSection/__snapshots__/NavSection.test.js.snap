// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavSection: default 1`] = `
<div
  className="sc-bdVaJa cXgeQK"
/>
`;
